src/components/safe-apps/SafeAppIcon/index.tsx

```diff
@@ -1,17 +1,19 @@
 import { type ReactElement, memo } from 'react'
 
-const getIframeContent = (src: string): string => {
-  return `<style>
-    html, body {
-      padding: 0;
-      margin: 0;
-      background-color: transparent;
-      height: 100%;
-    }
-    body {
-      background: url('${src}') center center/contain no-repeat;
-    }
-  </style>`
+const APP_LOGO_FALLBACK_IMAGE = `/images/apps/app-placeholder.svg`
+
+const getIframeContent = (url: string, width: number, height: number): string => {
+  return `
+     <body style="margin: 0;">
+       <img src="${url}" alt="App logo" width="${width}" height="${height}" />
```

Comment:
Like I said, you also might want to escape dangerous symbols here. `encodeURI(url)` should do the trick.

Reply:
The url is validated at the fetch time and also browsers seem to be smart nowadays: https://security.stackexchange.com/questions/124840/xss-vectors-in-img-src-and-background-image-url

Reply:
Given we have all these safety things in place I think an additional check inside the component is redundant

Reply:
It's not about fetching, you're inserting a string directly into HTML. It can close the tag and render a butt.

Reply:
Doesn't seem like a big security threat to me given that we validate the URL before and the amount of sandboxing we enforce on the iframe. But to get the pr through i added it